RecoEgamma/EgammaElectronProducers/python/gsfElectrons_cfi.py

```diff
@@ -13,9 +13,8 @@
     # input collections
```

Comment:
similar to the comment in gedGsfElectrons_cfi.py, it's a good opportunity to switch to using the cfi created by the fillDescriptions